app/utils/types.ts

```diff
@@ -106,19 +106,42 @@ export enum IdentityStatus {
 /**
  * This Interface models the structure of the identities stored in the database.
```

Comment:
This comment no longer matches the interface.

Reply:
I moved it down to Identity again.